src/content/docs/zh-cn/features/websocket.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Websocket
```

Comment:
```suggestion
title: WebSocket
```

Reply:
websocket is a proper term!  Does s need to be capitalized as well?